crowdsec-docs/unversioned/getting_started/healthcheck.mdx

```diff
@@ -0,0 +1,173 @@
+---
+id: health_check
+title: CrowdSec Security Engine Setup Health-Check
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+import { CheckboxProvider, InteractiveCheckbox } from '@site/src/components/InteractiveCheckbox.js';
+import HierarchicalList from '@site/src/components/HierarchicalList.js';
+
+Welcome to the interactive health check of your CrowdSec setup. We'll have a Top-Down approach:  
+First, check the proper functioning of the Security Engine's finality: **Detecting** AND **Protecting** against attacks.  
+Then, eventually, diving into troubleshooting any issue you may encounter.  
+
+## 📡 Detection checks
+
+*Trigger CrowdSec's test scenarios*  
+
+:::info
+Check that your Security Engine properly reads and parses the logs of the services you protect.  
+The HTTP collection and the Linux collection contain test scenarios allowing you to prove your Security Engine works as intended, without having to do penetration tests, risking getting you banned from your own server.
+:::
+
+### Test HTTP detection
+
+For this test, you'll add the following path to a URL of a website you're protecting: `/CRODWSEC/TEST`  
+_*// Any issue possible with .htaccess that would rewrite before this path reaches the log? – **Yes, this is a valid concern. Add a note: "Important: Ensure your web server configuration (e.g., .htaccess) does not rewrite or block this specific URL path `/CRODWSEC/TEST`."***_  
```

Comment:
- show the user how to do it with curl
- mention that if the test is done from a private IP you won't see it appear